pyopenms_viz/_core.py

```diff
@@ -430,6 +435,65 @@ def _kind(self):
         return "scatter"
 
 
+class SequencePlot(BasePlot, ABC):
```

Comment:
I am wondering if we should could create another class or broaden this one for compound plots? Similar to what we discussed with Hannes, for including generation of compound molecules using rdkit within pyopenms_viz?

Reply:
I think that we can create another class for CompoundPlot because I do not think there would be too much in common implementation wise with the SequencePlot class?

Reply:
Yeah, that makes sense :+1: